sites/partners/src/applications/PaperApplicationForm/sections/FormMultiselectQuestions.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useMemo } from "react"
+import {
+  Field,
+  t,
+  GridSection,
+  ViewItem,
+  GridCell,
+  FormAddress,
+  FieldGroup,
+} from "@bloom-housing/ui-components"
+
+import { useFormContext } from "react-hook-form"
+import { stateKeys, getInputType, fieldName } from "@bloom-housing/shared-helpers"
+import {
+  ApplicationSection,
+  ListingMultiselectQuestion,
+  MultiselectOption,
+  MultiselectQuestion,
+} from "@bloom-housing/backend-core/types"
+
+type FormMultiselectQuestionsProps = {
+  questions: ListingMultiselectQuestion[]
+  applicationSection: ApplicationSection
+  sectionTitle: string
+}
+
+const FormMultiselectQuestions = ({
+  applicationSection,
+  questions,
+  sectionTitle,
+}: FormMultiselectQuestionsProps) => {
+  const formMethods = useFormContext()
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, watch } = formMethods
+
+  const allOptionFieldNames = useMemo(() => {
+    const keys = []
+    questions?.forEach((listingQuestion) =>
+      listingQuestion?.multiselectQuestion.options.forEach((option) =>
+        keys.push(
+          fieldName(listingQuestion?.multiselectQuestion.text, applicationSection, option.text)
+        )
+      )
+    )
+
+    return keys
+  }, [questions, applicationSection])
+
+  if (questions?.length === 0) {
+    return <></>
```

Comment:
is this different than returning `null`?

Reply:
I don't believe so and null looks to be preferable in the docs, I'll update 👍